generators/app/templates/README.md

```diff
@@ -1,4 +1,4 @@
-# <%=testProjectName%>
+# myproject
```

Comment:
Please revert this. Yeoman Generator will put project name here, depending on what was passed as parameters